esda/moran.py

```diff
@@ -889,6 +875,9 @@ class Moran_Local(object):
         value to use as a weight for the "fake" neighbor for every island. If numpy.nan,
         will propagate to the final local statistic depending on the `stat_func`. If 0, then
         the lag is always zero for islands.
+    alternative: string
+        possible values -> "one-tailed"/ "two-tailed"
+        default value ->"two-tailed" 
```

Comment:
this part should contain a description of what the argument controls, i.e. what does it mean if I use `"one-tailed"` vs `"two-tailed"`. Does not have to be detailed but just a hint, an explanation in plain words. Like in `permutations`, where we say `"number of random permutations for calculation of pseudo p_values"`.